Mathlib/Algebra/Order/Antidiag/Pi.lean

```diff
@@ -74,5 +85,150 @@ where
 @[simp] lemma mem_finAntidiagonal {d : ℕ} {f : Fin d → μ} :
     f ∈ finAntidiagonal d n ↔ ∑ i, f i = n := (finAntidiagonal.aux d n).prop f
 
+/-!
+### `ι → μ`
+
+In this section, we transfer the antidiagonals in `Fin s.card → μ` to antidiagonals in `ι → s` by
+choosing an identification `s ≃ Fin s.card` and proving the end result does not depend on that
```

Comment:
```suggestion
choosing an identification `s ≃ Fin s.card` and proving that the end result does not depend on that
```
Also, can you say which is this result, and add to it (below) a bit of doc along the lines "this lemma/thm proves that ..."?